iofog-agent-daemon/src/main/java/org/eclipse/iofog/process_manager/ProcessManager.java

```diff
@@ -184,8 +185,11 @@ private void handleLatestMicroservices() {
 				Optional<Container> containerOptional = docker.getContainer(microservice.getMicroserviceUuid());
 
 				if (!containerOptional.isPresent() && !microservice.isDelete()) {
-					StatusReporter.setProcessManagerStatus().setMicroservicesState(microservice.getMicroserviceUuid(), MicroserviceState.QUEUED);
-					addMicroservice(microservice);
+					if (StatusReporter.getProcessManagerStatus().getMicroserviceStatus(microservice.getMicroserviceUuid()) != null
```

Comment:
These lines of code are too long, shorten by assigning to variables before final evaluation